.github/workflows/publish-theme.yaml

```diff
@@ -42,14 +42,9 @@ jobs:
                 uses: actions/setup-node@v3
                 with:
                     node-version: 16
-
-            -   name: Setup git user and npm
-                run: |
-                    git config --global user.name "Apify Release Bot"
```

Comment:
I would guess this needs to stay, we need to be able to push to git from here. Same with the NPM thingy, that is so we can publish new themes to the `@apify/` namespace.

Reply:
Ha, you're right. My bad, I thought it was connected only to the installation.